homeassistant/components/v2c/manifest.json

```diff
@@ -5,5 +5,5 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/v2c",
   "iot_class": "local_polling",
-  "requirements": ["pytrydan==0.7.0"]
+  "requirements": ["pytrydan==0.8.0"]
```

Comment:
Can we bump in a separate PR?

Reply:
https://github.com/home-assistant/core/pull/122898